templates/todo/projects/java-mongo-aca/.repo/bicep/azure.yaml

```diff
@@ -6,11 +6,9 @@ metadata:
 services:
```

Comment:
aren't these required to pass params to aca modules?

Reply:
In the new ACA model all infra would be provisioned up front and the service modules would be referenced from within the `main.bicep` modules.  We no longer need to reference these as part of the code deploy flow.